data/las.csv

```diff
@@ -1,167 +1,167 @@
-new_la_code,la_name,date_of_operation,date_of_termination,parent_geography_code,entity_code,status
```

Comment:
Should we add a column for the type of location - you had that file that explained what a E09 and E06 were, might be useful to have in here?

Reply:
I have entity_code in there which gives the E0- numbers? Would the description add anything? (I don't think they're included in the code history database)

Reply:
It's not a huge thing but I think the description would add something (fairly sure it's a trello card somewhere?). The entity_code feels redundant as you can just search the first 3 characters of the new_la_code to get the same result, whereas the description actually tells you what they mean. 

Happy for you to leave out of this PR if you don't have the file to hand.

Reply:
I've added the descriptions though I think the entity stuff may be confusing for anyone looking at it as I don't think many people realise LAs aren't really a thing in their own right.